lspci.h

```diff
@@ -179,12 +179,12 @@ void print_pci(struct pci p, const PCIDB & pciDB)
 
 void load_PCIDB(PCIDB & pciDB)
 {
-    std::ifstream in("pci.ids");
+    std::ifstream in("/usr/share/hwdata/pci.ids");
     std::string line, item;
 
     if (!in.is_open())
     {
-        std::cerr << "pci.ids file is not available. Download it from https://raw.githubusercontent.com/pciutils/pciids/master/pci.ids " << endl;
+        std::cerr << "/usr/share/hwdata/pci.ids file is not available. Ensure that the \"hwdata\" package is properly installed." << endl;
```

Comment:
please mention the old source of the pci.ids as well because on some distribution the package might be not available and/or outdated.  "Ensure that \"hwdata\" package is properly installed or download https://raw.githubusercontent.com/pciutils/pciids/master/pci.ids and copy it to /usr/share/hwdata/pci.ids"

Reply:
Done.

Reply:
Updated to consider FreeBSD, on which the path is /usr/local/share/pciids/pci.ids.